pkg/plugin/util/util.go

```diff
@@ -176,6 +176,48 @@ func UncommentCode(filename, target, prefix string) error {
 	return os.WriteFile(filename, out.Bytes(), 0644)
 }
 
+// CommentCode searches for target in the file and adds the comment prefix
+// to the target content. The target content may span multiple lines.
+func CommentCode(filename, target, prefix string) error {
```

Comment:
This one works exactly as the UncommentCode, however, doing the opposite so that we can disable the metrics or any other option now in our tests.